README.md

```diff
@@ -1,2 +1,12 @@
-# easyopac_status
-Integrate StatusPage.io API
+# EasyOPAC Status
+Integrate StatusPage.io API in order to record incidents occurred on the site.
+
+## Install
+Drupal standard module install.
+
+## Configuration
+Access the _/admin/reports/easyopac_status_ page. Populate the admin configs fieldset with the statuspage.io project credentials.
+API url by default is the _statuspage.io/api/v2/status.json_.
```

Comment:
New line:
"easySuite default Page ID is *mzgxvjlqnlzh*"

Reply:
Do I need to add this line? Will this Page ID be a default value?
Plus, this can be added as a placeholder for the admin page or even as a default value?